### ISSUE TITLE
As a Project Coordinator, I want to edit an operator, so that I can correct or add missing information

### ISSUE THREAD

**Issue by Sriram-pc** [bcgov/cas-cif#65]
#### Description:
As a CIF administrator, I want to edit an operator, so that I can edit the operator name to the project

#### Acceptance Criteria:

Given I am a CIF user and viewing the operators page
When I click on the "edit" button in the operators table
Then I am redirected to `cif/operator/<operatorId>/edit`

When that I am viewing the `cif/operator/<operatorId>/edit`
Then I see a form prepopulated with the existing operator information (excluding the swrsOrganisationId, which cannot be edited)

#### Development Checklist:
- [x] ensure that swrs_organisation_id is immutable
- [x] Operator edit follows same pattern as other forms, i.e. relying on the `form_change` table
- [x] legal name and trade name are required, bc_registry_id and operator_code are optional
- [x] Meets the DOD

#### Personas:
- [Program Development Team](https://bcgov.sharepoint.com/:b:/t/00608-CIFInternalUsers/EcVHGc4X8mhIvQHGSaGJmyAB9F1iBe7MSH1Y1fBKUc1skA?e=fzATIc)
- [Operations Team](https://bcgov.sharepoint.com/:b:/t/00608-CIFInternalUsers/EaswKcprmuNKpEuY29a-Os0BG-h5u-Ztou7UuzK3x0xfjw?e=e8oR1i)
- [Tech Team](https://bcgov.sharepoint.com/:b:/t/00608-CIFInternalUsers/Ec2AmUecYCdDlmURctGwTbMBUMGZXlUyyt5HQZJdpjeejA?e=0axXlm)

**Comment by Sriram-pc** [comment]
Adding applicable Personas to the ticket
[Program Development Team - CIF Persona](https://bcgov.sharepoint.com/:b:/t/00608-CIFInternalUsers/EcVHGc4X8mhIvQHGSaGJmyAB9F1iBe7MSH1Y1fBKUc1skA?e=fzATIc)

[Operations Team - Persona](https://bcgov.sharepoint.com/:b:/t/00608-CIFInternalUsers/EaswKcprmuNKpEuY29a-Os0BG-h5u-Ztou7UuzK3x0xfjw?e=e8oR1i)

**Comment by rajpersram** [comment]
![CIF Workflow Co-Design - Operator Page .jpg](https://images.zenhubusercontent.com/5c2e4cfdab4f3464fb8df8d1/e62f668f-4eb7-4d0c-9b07-b88076f7c4c8)

**Comment by rajpersram** [comment]
[See ticket 67](https://app.zenhub.com/workspaces/climate-action-secretariat-60ca4121764d710011481ca2/issues/bcgov/cas-cif/67) for wireframe

**Comment by Sriram-pc** [comment]
![image.png](https://images.zenhubusercontent.com/6149103e0c14630f5a9c158a/18edba1e-e7fa-4265-a1c4-e2c55fb6ad3c)

**Comment by LindsayMacfarlane** [comment]
Thank you @pbastia !

Tested in dev:
- swrs_id is immutable (isn't displayed on edit page)
- able to edit and update fields
- optional fields clearly indicated as such


Some observations:
- the fields are centered on the page. Other pages appear more left aligned. What is the best design practice?
- the error messages could be improved (see operator code example below)
![image.png](https://images.zenhubusercontent.com/60f9db089e3a388facde96d1/a7aff38f-37a4-4254-9112-773d6159a72f)
- When I removed the BC registry id and left null - that null value was not reflected in the operator table.

@suhafa - some future design considerations?
